pkg/plugin/windows/pktmon/pktmon_plugin_windows.go

```diff
@@ -130,27 +131,34 @@ func (p *Plugin) Start(ctx context.Context) error {
 		return ErrNilEnricher
 	}
 
-	go func() {
+	g, ctx := errgroup.WithContext(ctx)
+
+	g.Go(func() error {
 		err := p.RunPktMonServer(ctx)
 		if err != nil {
-			p.l.Error("pktmon server exited", zap.Error(err))
+			return fmt.Errorf("pktmon server exited: %w", err)
```

Comment:
[the errors package is deprecated](https://github.com/pkg/errors), are we keeping it around for the stack trace?

Reply:
i think i've had this exact conversation with @rbtr before 🙂

Reply:
Archived != deprecated, and "no longer being actively developed" is not a bad thing. We shouldn't abandon code because it's done enough that it hasn't needed to be updated in years.
On whether to use `pkg/errors` specifically: where both choices are comparably good, consistency is more important than anything else...since it's used everywhere already, I think we should keep using it. I also think the stacktraces are useful and prefer the ergonomics of `pkg/errors`, but you may disagree.
We could have a similar argrument with zap/logrus/klog/<etc> vs `slog`. I think zap has benefits, but you could argue for `slog` and it's comparable enough that it _would work_. But having consistency in style is even more important for logs.

Reply:
@rbtr I could dig up my `errlint` tool again and globally force it to be one way or another. We could add it to a pipeline and enforce it as well.

Reply:
I don't have plans to die on this hill either way, but if we are going to enforce this +1 to bringing the feedback loop closer and have `errlint` do the block earlier